backend/api/src/main/kotlin/software/shonk/adapters/incoming/ShorkInterpreterController.kt

```diff
@@ -48,6 +85,17 @@ fun Route.configureShorkInterpreterControllerV1() {
         return@get
     }
 
+    /**
+     * Creates a new lobby and returns the id of the newly created one, which identifies the lobby
+     * uniquely. If the lobby doesn't exist or the playerName is invalid, the create operation is
+     * aborted. The body must contain the desired playerName of the player creating the lobby. body:
```

Comment:
What do you mean with "if the lobby doesn't exist"? We are creating a new one, why should it exist already?